docs/install/install-redis/_index.md

```diff
@@ -1,159 +1,159 @@
 
-## Install Redis properly
+## Install Valkey properly
 
-Running Redis from the command line is fine just to hack a bit or for development. However, at some point you'll have some actual application to run on a real server. For this kind of usage you have two different choices:
+Running Valkey from the command line is fine just to hack a bit or for development. However, at some point you'll have some actual application to run on a real server. For this kind of usage you have two different choices:
 
-* Run Redis using screen.
-* Install Redis in your Linux box in a proper way using an init script, so that after a restart everything will start again properly.
+* Run Valkey using screen.
+* Install Valkey in your Linux box in a proper way using an init script, so that after a restart everything will start again properly.
 
 A proper install using an init script is strongly recommended. 
 
 {{% alert title="Note" color="warning" %}}
-The available packages for supported Linux distributions already include the capability of starting the Redis server from `/etc/init`.
+The available packages for supported Linux distributions already include the capability of starting the Valkey server from `/etc/init`.
 {{% /alert  %}}
 
-If you have not yet run `make install` after building the Redis source, you will need to do so before continuing. By default, `make install` will copy the `redis-server` and `redis-cli` binaries to `/usr/local/bin`.
+If you have not yet run `make install` after building the Valkey source, you will need to do so before continuing. By default, `make install` will copy the `valkey-server` and `valkey-cli` binaries to `/usr/local/bin`.
 
-* Create a directory in which to store your Redis config files and your data:
+* Create a directory in which to store your Valkey config files and your data:
 
     ```
-    sudo mkdir /etc/redis
-    sudo mkdir /var/redis
+    sudo mkdir /etc/valkey
+    sudo mkdir /var/valkey
     ```
 
-* Copy the init script that you'll find in the Redis distribution under the **utils** directory into `/etc/init.d`. We suggest calling it with the name of the port where you are running this instance of Redis. Make sure the resulting file has `0755` permissions.
+* Copy the init script that you'll find in the Valkey distribution under the **utils** directory into `/etc/init.d`. We suggest calling it with the name of the port where you are running this instance of Valkey. Make sure the resulting file has `0755` permissions.
     
     ```
-    sudo cp utils/redis_init_script /etc/init.d/redis_6379
+    sudo cp utils/redis_init_script /etc/init.d/valkey_6379
     ```
 
 * Edit the init script.
 
     ```
-    sudo vi /etc/init.d/redis_6379
+    sudo vi /etc/init.d/valkey_6379
     ```
 
 Make sure to set the **REDISPORT** variable to the port you are using.
```

Comment:
AFAICT, the init script `utils/redis_init_script` hasn't been renamed or rebranded within the code repo. When it's done, we'll add the label `needs-doc-pr` to fix the docs to match the reality. That's the usual process anyway. I didn't want to do this in the reverse order since stuff may be forgotten and not be correct then.

Reply:
And I didn't want to do any style fixes of the old docs. Just rebranding.

Reply:
Fair on the style fix and I get the order on the init script. That was surprising it hasn't been done yet.

Reply:
There's more to do in the main repo. Some log messages still contain "Redis" I believe.

The problem with the init script template is that nobody knows if it really works. It's not tested in CI. :) But we should rebrand it anyway.

Reply:
https://github.com/valkey-io/valkey/issues/354